### PR TITLE
Fix access on transit tube for gurfan

### LINKED ISSUE #31954: transit tubes are broken
(WEB REPORT BY: gurfan REMOTE: 206.221.180.138:7777)
# Revision
71c8bfcc3859029cc63c4192c579ef5df8f6b68b

You can't walk into them to enter the tubes anymore. Trying to drag yourself into them gives you a message saying 'access denied'

### PR DESCRIPTION
**Pull request by adacovsk**
Closes #31954
:cl:
 * tweak: fixed metaclub access on transit tubes

### CODE REVIEW

File: maps/metaclub.dmm [hunk 1]
```diff
@@ -42938,7 +42938,7 @@
 	},
 /obj/structure/transit_tube/station{
 	dir = 4;
-	req_access_txt = "32;19;501";
+	req_one_access_txt = "32;19;501";
```
Comment (Eneocho) [inline maps/metaclub.dmm:42941]: I feel the "_txt" part should not be there.

Comment (SonixApache) [reply maps/metaclub.dmm:42941]: @Eneocho you're wrong, they should be

Comment (Eneocho) [reply maps/metaclub.dmm:42941]: I don't know shit about accesscode, but I've never seen "req_one_access_txt" before. Only "req_access_txt" or "req_one_access".

Comment (gurfan) [reply maps/metaclub.dmm:42941]: That would be because you don't know shit about accesscode

Comment (Eneocho) [reply maps/metaclub.dmm:42941]: Precisely what I said.

**Pull request merged**